fakestorage/server_test.go

```diff
@@ -13,6 +13,8 @@ import (
 	"strings"
 	"testing"
 
+	"google.golang.org/api/iterator"
+
```

Comment:
can you delete this line? (I realize it's something goimports add, but it's annoying lol)